client/src/components/HomePage.tsx

```diff
@@ -76,8 +89,57 @@ const HomePage = withRouter((props: any) => (
           }}
         </Mutation>
       )}
-    </UserContext.Consumer>
+          </UserContext.Consumer>
+      </Grid.Row>
+      <Grid.Row style={{
+        height: "50%",
+        backgroundColor: "white",
+        }}>
+        <div>
+    <Grid columns={2} padded style={{
+      paddingLeft: '50px',
+      paddingRight: '100px',
+      }}>
+      <Grid.Column style={{
+        paddingTop: '100px',
+        fontStyle: 'Sans-serif',
+        fontSize: '17px',
+      }}>
+      <p style={{
+        fontSize: '20px',
+        fontStyle: 'bold',
+      }}><b>Our Mission</b></p>
+        <p> Provide a streamlined, centralized, and secure platform
+        for the student organization recruiting process.</p>
+      </Grid.Column>
+      <Grid.Column style={{
+        paddingTop: '50px',
+        paddingLeft: '200px',
+        paddingBottom: '50px'}}>
+      <img src={require('/usr/src/app/src/images/cow.png')}
```

Comment:
The images should be `import`ed at the top but more importantly, you haven't included them in your PR

Reply:
Actually require might be the right way to do it, if it works

Reply:
My bad, didn't see the images. Just need to update the image links to relative